joomla/joomla-cms
# Converts none editor to service providers

### Summary of Changes
Converts the none editor to service providers.

This pr contains the commits from #39625.

### Testing Instructions
- Make sure None is the default editor
- Edit an article

Also test if the other editors do still work

### Actual result BEFORE applying this Pull Request
All works.

### Expected result AFTER applying this Pull Request
All works.

### Link to documentations
Please select:
